rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/20_terms.yml

```diff
@@ -695,6 +695,36 @@ setup:
 
   - match: { indices.test_1.total.fielddata.memory_size_in_bytes: 0}
 
+---
+"Global ordinals are loaded with the global_ordinals execution hint":
+
+  - do:
+      index:
+        refresh: true
+        index: test_1
+        id: 1
+        routing: 1
+        body: { "str": "abc" }
+
+  - do:
+      index:
+        refresh: true
+        index: test_1
+        id: 2
+        routing: 1
+        body: { "str": "abc" }
+
+  - do:
+      index:
+        refresh: true
+        index: test_1
+        id: 3
+        routing: 1
+        body: { "str": "bcd" }
+
+  - do:
+      indices.refresh: {}
+
```

Comment:
I see only one test added here, so I don't follow what is being split. Also, would it make sense to make the test changes to master directly?

Reply:
I'm cutting this test in half.

I think it'd make sense to do this in master if we expected folks to hack on these tests before we merge this down. I'm mostly the person who I'd expect to hack on these tests so I don't think that is super likely.